<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Kutta</class>
 <widget class="QDialog" name="Kutta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Initial point (Xi)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Final point (Xf)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Increment costant/Slope of fuction (h)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QDoubleSpinBox" name="Xi2_spinbox"/>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="Xf2_spinbox"/>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="h2_spinbox"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="METHOD_BOTTOM">
       <property name="text">
        <string>METHOD</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ERASE2_BOTTON">
       <property name="text">
        <string>ERASE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>CLOSE</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
